R/ppmc.R

```diff
@@ -55,26 +56,30 @@
 #' @return A list with two elements, "model_fit" and "item_fit". If either
 #'   `model_fit = NULL` or `item_fit = NULL` in the function call, this will be
 #'   a one-element list, with the null criteria excluded. Each list element, is
-#'   itself a list with one element for each specified PPMC containing a
-#'   [tibble][tibble::tibble-package]. For example if
+#'   itself a list with one element for each specified \acronym{PPMC} containing
+#'   a [tibble][tibble::tibble-package]. For example if
 #'   `item_fit = c("conditional_prob", "odds_ratio")`, the "item_fit" element
 #'   will be a list of length two, where each element is a tibble containing the
-#'   results of the PPMC. All tibbles follow the same general structure:
+#'   results of the \acronym{PPMC}. All tibbles follow the same general
+#'   structure:
 #'
-#'   * `obs_{ppmc}`: The value of the relevant statistic in the observed data.
-#'   * `ppmc_mean`: The mean of the `ndraws` posterior samples calculated for
-#'     the given statistic.
-#'   * Quantile columns: 1 column for each value of `probs`, providing the
+#'   \itemize{
+#'   \item{`obs_{ppmc}`: }{The value of the relevant statistic in the observed
+#'     data.}
+#'   \item{`ppmc_mean`: }{The mean of the `ndraws` posterior samples calculated
+#'     for the given statistic.}
+#'   \item{Quantile columns: }{1 column for each value of `probs`, providing the
```

Comment:
This might be a controversial one, but the default `roxygen2` parsing of the bulleted lists will treat this markdown like an entire text block. 

By contrast, `\itemize{\item{<name>}{<description>}}` let us separate the term which should be ignored by the linter from the description that should be linted.